<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyRandrWidget</class>
 <widget class="QWidget" name="PyRandrWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyRandrWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_instruction">
       <property name="text">
        <string>Select which region of the screen you'd like to create a virtual display for:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_full_screen">
       <property name="text">
        <string>Select Full Screen</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_select_region">
       <property name="text">
        <string>Select Region of Screen</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_box_select_half">
       <property name="title">
        <string>Select Half of Screen</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontal_layout_select_half">
          <item>
           <widget class="QPushButton" name="push_button_left_half">
            <property name="text">
             <string>Left</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="push_button_right_half">
            <property name="text">
             <string>Right</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select Third of Screen</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="push_button_left_third">
            <property name="text">
             <string>Left</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="push_button_center_third">
            <property name="text">
             <string>Center</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="push_button_right_third">
            <property name="text">
             <string>Right</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_box_select_quarter">
       <property name="title">
        <string>Select Quarter of Screen</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="push_button_top_left_quarter">
          <property name="text">
           <string>Top Left</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="push_button_top_right_quarter">
          <property name="text">
           <string>Top Right</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="push_button_bottom_left_quarter">
          <property name="text">
           <string>Bottom Left</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="push_button_bottom_right_quarter">
          <property name="text">
           <string>Bottom Right</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_box_select_sixth">
       <property name="title">
        <string>Select Sixth of Screen</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <layout class="QGridLayout" name="grid_layout_select_sixth">
          <item row="1" column="0">
           <widget class="QPushButton" name="push_button_top_left_sixth">
            <property name="text">
             <string>Top Left</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="push_button_top_center_sixth">
            <property name="text">
             <string>Top Center</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="push_button_bottom_right_sixth">
            <property name="text">
             <string>Bottom Right</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="push_button_bottom_left_sixth">
            <property name="text">
             <string>Bottom Left</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="push_button_bottom_center_sixth">
            <property name="text">
             <string>Bottom Center</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="push_button_top_right_sixth">
            <property name="text">
             <string>Top Right</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
